quiz/backend/players/tests/test_views.py

```diff
@@ -0,0 +1,15 @@
+from players.views import PlayerCreateAPIView
+
+
+def test_player_view_return_201_status_code_and_correct_data(player_api):
+    response = player_api()
+
+    assert response.status_code == 201
+    assert response.data["nick"] == "test_nick"
```

Comment:
"test_nick" could be generated by fixture and used by test and fixture generating data.